src/pypromice/process/aws.py

```diff
@@ -224,7 +224,7 @@ def loadL0(self):
 
             except pd.errors.ParserError as e:
                 # ParserError: Too many columns specified: expected 40 and found 38
-                logger.info(f'-----> No msg_lat or msg_lon for {k}')
+                # logger.info(f'-----> No msg_lat or msg_lon for {k}')
```

Comment:
msg_lat and msg_lon are no longer printed in the output files because deemed unreliable (#199)
They are still scraped from the SBD messages and appended to some, but not all transmission files. In the future we should stop scraping them, but in the meantime, no need to warn anyone when they are not found in transmission files.